libs/components/src/lib/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,100 @@
+import { FoundationElement } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
+
+/**
+ * Base class for accordion-item
+ *
+ * @public
+ */
+export class accordionItem extends FoundationElement {
+	/**
+	 * 
+	 *
+	 * @public
+	 * 
+	 * HTML Attribute: heading
+	 */
+	@attr({ mode: 'fromView' }) heading = '';
+
+	/**
+	 *
+	 * @public
+	 * HTML Attribute: heading-level
+	 */
+	@attr({ mode: 'fromView', attribute: 'heading-level' }) headingLevel: 2 | 3 | 4 | 5 | 6 = 3;
```

Comment:
no need to assign value. just default to 3 if no assigned value
```suggestion
	@attr({ attribute: 'heading-level' }) headingLevel: 2 | 3 | 4 | 5 | 6;
```